doc/cprover-manual/cbmc-assertions.md

```diff
@@ -87,11 +87,19 @@ postconditions to create a function contract, which can be used for
 modular verification.
 
 
-Future CPROVER releases will support explicit quantifiers with a syntax
+CPROVER also supports explicit quantifiers with a syntax
 that resembles Spec\#:
 
 ```C
 __CPROVER_forall { *type identifier* ; *expression* }
 __CPROVER_exists { *type identifier* ; *expression* }
 ```
 
+For example, the following can be used to check equality of
+two arrays `array` and `array2` to a bound `n`.
```

Comment:
is there a limit on the bound `n`? I recall at one point it only worked up to 100?